packages/access/src/accesscontrol/accesscontrol.cairo

```diff
@@ -31,16 +42,16 @@ pub mod AccessControlComponent {
 
     /// Emitted when `account` is granted `role`.
     ///
-    /// `sender` is the account that originated the contract call, an admin role
-    /// bearer (except if `_grant_role` is called during initialization from the constructor).
+    /// `sender` is the account that originated the contract call, an account with the admin role
+    /// or the zero address if `grant_role` is called from the constructor.
```

Comment:
```suggestion
    /// or the deployer address if `grant_role` is called from the constructor.
```

The caller in constructor doesn't have to be zero. It is only zero in deploy_account transactions but not when using syscalls.